src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/_help.py

```diff
@@ -99,7 +99,7 @@
 
 helps['acr login'] = """
     type: command
-    short-summary: Log in to a container registry through Docker.
+    short-summary: Log in to a container registry through docker.
```

Comment:
I believe Docker should remain capitalized.  @sptramer?

Reply:
I thought `Docker` is a trademark while we actually meant the docker CLI here.

Reply:
In that case can we make that explicit?

Reply:
Yes, if this is meant to be the CLI (especially if it relies on the Docker CLI being locally installed) then it should be `the Docker CLI`, `the docker command` or whatever other language makes the most sense here. `long-summary` may need to include information such as `Docker must be installed on your machine` or any other pre-conditions to getting this command working.